lib/mshoplib/src/MShop/Coupon/Provider/Decorator/Category.php

```diff
@@ -71,22 +71,32 @@ public function calcPrice( \Aimeos\MShop\Order\Item\Base\Iface $base ) : \Aimeos
 			$filter = $manager->filter( true )->add( ['catalog.code' => $codes] )->slice( 0, count( $codes ) );
 			$catIds = $manager->search( $filter )->keys()->toArray();
 
-			$filter = $listManager->filter( true )->slice( 0, count( $prodIds ) )->add( [
+			$filter = $listManager->filter( true )->add( [
 				'catalog.lists.parentid' => $catIds,
 				'catalog.lists.refid' => array_keys( $prodIds ),
 				'catalog.lists.type' => ['default', 'promotion'],
 				'catalog.lists.domain' => 'product',
-			] );
+			] );	
```

Comment:
There's a tab at the end of the line which causes the coding style check to fail